tests/pytorch/distributed/run_gemm_with_overlap.py

```diff
@@ -505,7 +464,7 @@ def dist_print(msg, src=None, info=False, error=False, section=False, group=None
     else:
         ref_g = torch.matmul(inp_g, ker_g)
         if ub_obj2 is not None:
-            inp2_g = torch.nn.functional.gelu(ref_g)
+            inp2_g = torch.nn.functional.gelu(ref_g)  # pylint: disable=not-callable
```

Comment:
Is this suppression needed?
```suggestion
            inp2_g = torch.nn.functional.gelu(ref_g)
```

Reply:
Unfortunately yes. Everything in `torch.nn.functional` is generated by an `_add_docstring()` wrapper defined in `torch._C.__init__.pyi.in`, but [PyLint does not parse `*.pyi` files](https://github.com/pytorch/pytorch/issues/119482) so it does not understand that this wrapper preserves the callable property of the wrapped PyObject.